Containerfile-centos-arm64

```diff
@@ -0,0 +1,17 @@
+ARG version=stream9
+
+FROM quay.io/centos/centos:$version-minimal as builder
+ARG name=CentOS
+ARG arch=aa64
+ARG baseurl=https://mirror.stream.centos.org
+WORKDIR /root
+
+# Artifacts from kickstart repository.
+RUN . /etc/os-release && curl --location --remote-name-all --remote-time \
+	$baseurl/${VERSION}-stream/BaseOS/$(uname -m)/os/images/pxeboot/vmlinuz \
```

Comment:
VERSION will contain stream9, I suppose?

Reply:
No, 9. It's a bit confusing that `version` is the parameter and `VERSION` comes from `/etc/os-release`. Lower I also use the same trick to get the correct value `centos` in `/boot/efi/EFI/${ID}/{shim,grub}${arch}.efi`
```console
# . /etc/os-release && echo $VERSION
9
# . /etc/os-release && echo $ID
centos
```

I thought it was easier than to use shell to mangle `9stream` into `9`.

Reply:
ah gotcha, makes sense

Reply:
Yeah nice trick, I wonder if there is any chance of having a common file for all platforms.